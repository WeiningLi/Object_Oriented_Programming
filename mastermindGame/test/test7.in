ABCB
AAAA
DDDD
EEEE
FFFF
EEEE
CCCC
DDAA
BABA
BEFD
N
