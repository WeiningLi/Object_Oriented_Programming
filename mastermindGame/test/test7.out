Welcome to Mastermind! Please enter your first guess:
2b,1w
9 guesses left. Enter guess:
1b,0w
8 guesses left. Enter guess:
0b,0w
7 guesses left. Enter guess:
0b,0w
6 guesses left. Enter guess:
1b,0w
5 guesses left. Enter guess:
0b,0w
4 guesses left. Enter guess:
0b,0w
3 guesses left. Enter guess:
0b,1w
2 guesses left. Enter guess:
1b,2w
1 guesses left. Enter guess:
0b,2w
You lost! The password was: AFBB
Would you like to play again? (Y/N): 
